starter-code/src/functions-and-arrays.js

```diff
@@ -64,9 +87,21 @@ const wordsCount = [
   'matter',
   'truth',
   'disobedience',
-  'matter'
+  'matter',
 ];
 
+function howManyTimes(words, word) {
+  let newArr = [];
+  let count = 0;
+  for (let i = 0; i < words.length; i++) {
+    newArr.push(words[i].includes(word));
+    if (newArr[i] == true) {
+      count++;
```

Comment:
Interesting approach. You could have directly checked if its true to then add it to the newArr. And then at the end just return newArr.leength